portfolio/src/main/webapp/comments.js

```diff
@@ -44,32 +44,53 @@ async function getCommentsList() {
 
 // HELPER_FUNCTIONS
 /** 
- * adds an <li> element containing @param text to the HTML Element @param list.
- * @param {String} text
+ * adds an <div> element containing @param commentObj to the HTML Element @param list.
+ * @param {Object} commentObj
  * @param {HTMLElement} list
  */
 function addToList(commentObj, list) {
-  const comment = document.createElement('li');
-  comment.innerText = commentObj.text;
+  const rowElement = createRowElement();
+  const commentElement = createCommentElement(commentObj.text);
+  const deleteButtonElement = createDeleteButtonElement(commentObj)
+  rowElement.appendChild(commentElement)
+  rowElement.appendChild(deleteButtonElement);
```

Comment:
Nit (wtf/nit): Ideally create variables as close as possible to where they are used, especially if they are only used once. If I read through this code as-is, I read about `commentElement`, then I read about `deleteButtonElement`, then I have to ignore what `deleteButtonElement` does and remember what `commentElement` does so that I can know what `rowElement.appendChild(commentElement)` does.

This is harder than just figuring out what `commentElement` is, then seeing that it is added to `rowElement` then moving on to figure out what `deleteButtonElement` does and seeing that it is added to `rowElement`.

Also, you're missing semicolons at the ends of 2 of these lines.

```suggestion
  const commentElement = createCommentElement(commentObj.text);
  rowElement.appendChild(commentElement);
  const deleteButtonElement = createDeleteButtonElement(commentObj);
  rowElement.appendChild(deleteButtonElement);
```